build_mobilenet_base.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+if [ -z $1 ]; then
+    echo "Please specify wasmedge path"
+else
+    WASMEDGE=$1
+    WASI_NN_DIR=$(dirname "$0" | xargs dirname)
+    WASI_NN_DIR=$(realpath $WASI_NN_DIR)
+    WASMEDGE=$(realpath $WASMEDGE)
+    source /opt/intel/openvino_2021/bin/setupvars.sh
+
+    FIXTURE=https://github.com/intel/openvino-rs/raw/main/crates/openvino/tests/fixtures/mobilenet
+    pushd $WASI_NN_DIR/rust/
+    mkdir -p $WASI_NN_DIR/rust/examples/mobilenet-base/build
+    RUST_BUILD_DIR=$(realpath $WASI_NN_DIR/rust/examples/mobilenet-base/build/)
+    pushd examples/mobilenet-base
+    cargo build --release --target=wasm32-wasi
+    cp target/wasm32-wasi/release/mobilenet-base-example.wasm $RUST_BUILD_DIR
+    pushd build
+
+    if [ ! -f $RUST_BUILD_DIR/mobilenet.bin ]; then
+        wget --no-clobber --directory-prefix=$RUST_BUILD_DIR $FIXTURE/mobilenet.bin
+    fi
+    if [ ! -f $RUST_BUILD_DIR/mobilenet.xml ]; then
+        wget --no-clobber --directory-prefix=$RUST_BUILD_DIR $FIXTURE/mobilenet.xml
+    fi
+    if [ ! -f $RUST_BUILD_DIR/tensor-1x224x224x3-f32.bgr ]; then
+        wget --no-clobber $FIXTURE/tensor-1x224x224x3-f32.bgr --output-document=$RUST_BUILD_DIR/tensor-1x224x224x3-f32.bgr
+    fi
+    # Manually run .wasm
+    echo "Running example with WasmEdge ${WASMEDGE}"
+    $WASMEDGE --dir fixture:$RUST_BUILD_DIR mobilenet-base-example.wasm "fixture/mobilenet.xml" "fixture/mobilenet.bin" "fixture/tensor-1x224x224x3-f32.bgr"
+fi
```

Comment:
I think the fixtures are actually not related to the wasm file?
The program is trying to load the fixtures from a common folder `fixture/`, which pretty much just like your example, but with a  `fixture/` prefix.
The wasm file can load fixtures from any path, as long as you bind the path to `fixture` like this
```
$WASMEDGE --dir fixture:$FIXTURE_PATH...
```

Reply:
I think so too.
But I tested with `$WASMEDGE --dir fixture:$FIXTURE_PATH...` but failed before.
I don't know the reason and skip it for testing rapidly with your script instead.
Can you test with the command `$WASMEDGE --dir .:. mobilenet.xml mobilenet.bin tensor-1x224x224x3-f32.bgr` and ensure that it can be successfully run in your example?
Thanks.

Reply:
Maybe the error was caused by something else? I tried and seemed all good:
```
$ /root/WasmEdge/build/tools/wasmedge/wasmedge --dir .:. mobilenet-base-example.wasm "mobilenet.xml" "mobilenet.bin" "tensor-1x224x224x3-f32.bgr"
Read graph XML, size in bytes: 143524
Read graph weights, size in bytes: 13956476
Loaded graph into wasi-nn with ID: 0
Created wasi-nn execution context with ID: 0
Read input tensor, size in bytes: 602112
Executed graph inference
   1.) [963](0.7113)pizza, pizza pie
   2.) [762](0.0707)restaurant, eating house, eating place, eatery
   3.) [909](0.0364)wok
   4.) [926](0.0155)hot pot, hotpot
   5.) [567](0.0153)frying pan, frypan, skillet
```